.vscode/c_cpp_properties.json

```diff
@@ -8,7 +8,10 @@
                 "${workspaceFolder}/include",
```

Comment:
Also, given that some of the included files are now nested in include/at_chops, does this need to change? Or does vscode already recursively check nested folders?

Reply:
Good point, will investigate.

Reply:
https://github.com/atsign-foundation/at_c/pull/7/commits/9d868a803b2bf589bcf17137e55c16a9a0c29137 done